spark/src/main/scala/org/apache/comet/serde/QueryPlanSerde.scala

```diff
@@ -769,11 +769,7 @@ object QueryPlanSerde extends Logging with ShimQueryPlanSerde with CometExprShim
         val numBitsExpr = exprToProto(numBits, inputs, binding)
         val dataType = serializeDataType(bloom_filter.dataType)
 
-        // TODO: Support more types
-        //  https://github.com/apache/datafusion-comet/issues/1023
         if (childExpr.isDefined &&
-          child.dataType
-            .isInstanceOf[LongType] &&
```

Comment:
Rather than remove the type check, should we update the check for the supported types, or are we confident that we support all the types and that future versions of Spark won't support additional types?

Reply:
+1